docs/src/main/asciidoc/security-jwt-build.adoc

```diff
@@ -55,117 +58,144 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
 
-Note the `sign` step can be combined with the <<encrypt-claims,encrypt>> step to produce `inner-signed and encrypted` tokens, see <<innersign-encrypt-claims,Sign the claims and encrypt the nested JWT token>> section.
+You can combine the `sign` step with the <<encrypt-claims,encrypt>> step to create `inner-signed and encrypted` tokens. For more information, see the <<innersign-encrypt-claims,Sign the claims and encrypt the nested JWT token>> section.
 
 [[encrypt-claims]]
 == Encrypt the claims
 
-The claims can be encrypted immediately or after the `JSON Web Encryption` headers have been set the same way as they can be signed.
-The only minor difference is that encrypting the claims always requires a `jwe()` `JwtEncryptionBuilder` transition given that the API has been optimized to support signing and inner-signing of the claims.
+You can encrypt claims immediately or after setting the `JSON Web Encryption (JWE)` headers, similar to how claims are signed.
+However, encrypting claims always requires a `jwe()` transition to a `JwtEncryptionBuilder` because the API is optimized to support signing and inner-signing operations.
 
 [source, java]
 ----
 import io.smallrye.jwt.build.Jwt;
 ...
 
-// Encrypt the claims using an RSA public key loaded from the location set with a 'smallrye.jwt.encrypt.key.location' property. Default key encryption algorithm is RSA-OAEP.
+// Encrypt the claims using an RSA public key loaded from the location specified by the 'smallrye.jwt.encrypt.key.location' property.
+// The default key encryption algorithm is RSA-OAEP.
+
 String jwt1 = Jwt.claims("/tokenClaims.json").jwe().encrypt();
 
-// Set the headers and encrypt the claims with an RSA public key loaded in the code (the implementation of this method is omitted).  Default key encryption algorithm is A256KW.
+// Set the headers and encrypt the claims by using an RSA public key loaded in the code (the implementation of this method is omitted).
+// The default key encryption algorithm is A256KW.
 String jwt2 = Jwt.claims("/tokenClaims.json").jwe().header("custom-header", "custom-value").encrypt(getSecretKey());
 ----
 
-Note the `alg` (key management algorithm) header is set to `RSA-OAEP` and the `enc` (content encryption header) is set to `A256GCM` by default.
+Default behaviors:
+
+- The `alg` (key management algorithm) header defaults to `RSA-OAEP`.
+- The `enc` (content encryption) header defaults to `A256GCM`.
 
-RSA and Elliptic Curve (EC) public keys as well as symmetric secret keys can be used to encrypt the claims.
-`ECDH-ES` and `A256KW` are the default algorithms for EC public and symmetric key encryption algorithms respectively.
+Supported keys and algorithms:
+
+- You can use RSA public keys, Elliptic Curve (EC) public keys, and symmetric secret keys, to encrypt the claims.
+- `RSA-OAEP` is the default algorithm for the RSA.
+- `ECDH-ES` is the default algorithm for the EC public keys.
+- `A256KW` is the default algorithm for the symmetric secret keys.
```

Comment:
```suggestion
- `A256KW` is the default algorithm for the symmetric key encryption algorithms.
```

This it equivalent now to what is said in the replaced sentence

Reply:
Thank you. Updated per our Slack conversation. Please let me know if you have any further suggestions or feedback.